python/packages/ai/teams/ai/prompts/completion_config.py

```diff
@@ -58,6 +60,15 @@ class CompletionConfig:
         top_p (float): The model's top_p as a number between 0 and 2.
           Defaults to 0.
 
+        include_tools (Optional[bool]): If True, function calling will be enabled with the LLM.
+          Defaults to False.
+
+        tool_choice (Optional[chat.ChatCompletionToolChoiceOptionParam]):
```

Comment:
I don't think we need to include these, since we are using `actions.json` as a way to map our actions to send to the model.

Reply:
`tool_choice` is one of the [params](https://platform.openai.com/docs/api-reference/chat/create#chat-create-tool_choice) sent to the API call, so we want the customer to be able to define this. Did you by chance mean `include_tools`? That we are using to allow enabling/disabling tools, but with the logic being put into `ToolsAugmentation` instead of the model, I can see why that might be unnecessary - if they have actions and it's a tools augmentation, we can assume they want to use server-side tools. Thoughts?

Reply:
yes I meant for `include_tools` which is not part of the api. enabling/disabling of tools functionality whould be configured by using `augmentation_type: "none"` instead of tools augmentation.